openhtf/io/proto/test_runs.proto

```diff
@@ -17,9 +17,41 @@ syntax = "proto2";
 
 package openhtf;
 
+import "assembly.proto";
```

Comment:
The purpose of this is to, when importing to google3, drop these protos entirely and link against the internal ones. Though I may go with the other internal proto that's being made now so I'll just open-source that later rather than do this process multiple times.